code/backend/pages/01_Ingest_Data.py

```diff
@@ -29,6 +32,24 @@
 st.markdown(mod_page_style, unsafe_allow_html=True)
 
 
+def reprocess_all():
+    if env_helper.AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION:
+        reprocess_integrated_vectorization()
+    else:
+        remote_convert_files_and_add_embeddings()
+
+
+def reprocess_integrated_vectorization():
+    azure_search_indexer = AzureSearchIndexer(env_helper)
```

Comment:
I'm just wondering if this should be done in the backend functions? I think our pattern is normally `Admin` -> `Backend Function` for ingesting into AI Search.

Reply:
So we create a new Http trigger function? Similar to `AddURLEmbeddings` ?

Reply:
I think we could add it to the `BatchStartProcessing` function. We'll need to move the fork of if IV is enabled or not in there instead.